lib/ueberauth/strategy/twitter.ex

```diff
@@ -124,4 +124,8 @@ defmodule Ueberauth.Strategy.Twitter do
     |> options
     |> Keyword.get(key, default)
   end
+
+  defp json_library do
+    Ueberauth.Config.json_library() || Jason
```

Comment:
Sorry, my mistake by using words out of context.

We don't need `Jason` here since it will be by default on `Ueberauth.Config.json_library/0` when I said `Jason` by default I was referring to `Ueberauth.Config.json_library/0`

Reply:
```suggestion
    Ueberauth.Config.json_library()
```